hever/style.css

```diff
@@ -181,6 +175,43 @@ body {
  * Redefine Sass map values for child theme output.
  * - See: style-child-theme.scss
  */
+/**
+ * Redefine Sass map values for child theme output.
+ * - See: style-child-theme.scss
+ */
+/**
+ * Global
```

Comment:
This is really ugly but I need to import Varia's variable file in order to merge it with the child's. We could change the comments to not be multiline (I think those don't carry over to the compiled css) or just leave it like it is.

Reply:
Fixed!